tests/e2e.rs

```diff
@@ -661,29 +695,110 @@ async fn test_e2e_sync_deactivated_only() {
 	}
 }
 
+#[test(tokio::test)]
```

Comment:
I think it'd be nicer to have a `tests/e2e/ukt.rs` and a `tests/e2e/ldap.rs`.

Reply:
As per your next comments, first by adding a test that combines LDAP and UKT syncing into one, this separation is not necessary at this point; and secondly, as you mention later, splitting test groups can be part of another PR if necessary.